js/webui/webpack.config.js

```diff
@@ -21,13 +21,25 @@ function configCommon(config, params)
     });
 
     config.module.rules.push({
-        test: /(\.svg|\.png)$/,
+        test: /(\.svg|\.png|\.mp3)$/,
         loader: 'url-loader',
         options: {
             name: '[name].[ext]',
             limit: 1024
         }
     });
+
+    config.optimization = {
```

Comment:
This looks unrelated to this PR, besides that I'm not sure if there is a benefit of having vendors chunk.

Reply:
It's out of scope of this PR, but the added code pushed the bundle over 300KB, breaking the build

Reply:
The limit is raised in [other PR](https://github.com/hyperblast/beefweb/pull/184/commits/1145861ea8121191d0dbd9d5d9bcebb77dee1328), probably you can do the same.

Reply:
Removed code splitting and raised bundle limit